integration/vehicle-inspection/src/server/index.ts

```diff
@@ -0,0 +1,116 @@
+import {Logger} from '@raincatcher/logger';
+import {
+  BaseTask,
+  Process,
+  ProcessImpl,
+  ProcessInstance,
+  ProcessInstanceRepository,
+  ProcessRepository,
+  Task,
+  TaskRepository
+} from '@raincatcher/wfm';
+import * as Promise from 'bluebird';
+import * as bodyParser from 'body-parser';
+import * as express from 'express';
+import {isFunction, map} from 'lodash';
+import {VehicleInspectionTask} from '../vehicle-inspection/VehicleInspectionTask';
+
+export class Server {
+  public app = express();
+  public port = process.env.PORT || 3000;
+
+  constructor(
+    protected processRepo: ProcessRepository,
+    protected processInstanceRepo: ProcessInstanceRepository,
+    protected logger: Logger
+  ) {
+    this.setupExpressMiddleware();
+    this.setupExpressRoutes();
+  }
+
+  public listen(cb?: () => any) {
+    return this.app.listen(this.port, () => {
+      this.logger.info(`Server listening at port ${this.port}`);
+      if (isFunction(cb)) {
+        cb();
+      }
+    });
+  }
+
+  protected setupExpressMiddleware() {
+    this.app.use(bodyParser.json());
+    this.app.use(bodyParser.urlencoded({extended: true}));
+    // TODO: Add new security module
+  }
+
+  protected setupExpressRoutes() {
+    this.app.use('/processes', this.buildProcessRouter());
+    this.app.use('/instances', this.buildProcessInstanceRouter());
+    this.app.use('/tasks', this.buildTaskRouter());
+  }
+
+  protected buildProcessRouter() {
```

Comment:
Awesome! Is that for portal usage? Looks like you nailed that already. Not sure if that's actually serving purpose of this integration tests but we can move that out to actual cloud app or separate module!